### PR TITLE
Add Triggers v0.27.x  release

### PR DESCRIPTION
**Pull request by khrm**
# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/website/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Review by khrm — COMMENTED** [review]
/kind misc

**Comment by khrm** [comment]
Not sure why it's failing tests.

**Review by AlanGreene — APPROVED** [review]
Thanks @khrm 

~~I don't see v0.26 in https://github.com/tektoncd/triggers/blob/main/releases.md but assume it was a regular release (i.e. not LTS).~~

EDIT: I see you have https://github.com/tektoncd/triggers/pull/1729 to document v0.26 as an LTS release.

This means v0.23 and v0.25 are EOL and should be removed from the list so that we only provide documentation on the website for currently supported releases.

Based on the comments in the PR above I assume v0.27 is also LTS. Can you update the display names for both to mark them as LTS for users? (i.e. v0.27.x-LTS and v0.26.x-LTS)?

It would also be really helpful to include 'LTS' in the GitHub release names to make this obvious.

Thanks

**Comment by khrm** [comment]
@AlanGreene 26 was LTS. Doc PR isn't merged. Let me check what are the supported releases.

File: sync/config/triggers.yaml [hunk 1]
```diff
@@ -227,11 +48,8 @@ tags:
       target: getting-started
       include:
       - create-ingress.yaml
-      - create-webhook.yaml
-- name: v0.9.1
-  # The name to display on tekton.dev.
-  # sync.py will use this value in the version switcher and other places.
-  displayName: v0.9.1
+- name: release-v0.25.x
+  displayName: v0.25.x-LTS
```
Comment (khrm) [inline sync/config/triggers.yaml:52]: Added LTS to older releases also.

File: sync/config/triggers.yaml [hunk 2]
```diff
@@ -26,159 +26,8 @@ archive: https://github.com/tektoncd/triggers/tags
...
-  # Dict of folders to sync
-  folders:
-    docs:
-      index: README.md
-      include: ['*.md']
-    docs/getting-started:
-      target: getting-started
-      include:
-      - create-ingress.yaml
-- name: release-v0.16.x
```
Comment (khrm) [inline sync/config/triggers.yaml:139]: I have removed these older releases. @AlanGreene

**Pull request merged**